### PR TITLE
build: various make improvements

### PR DESCRIPTION
**Pull request by benesch**

### CODE REVIEW

File: build/protobuf.mk [hunk 1]
```diff
@@ -62,12 +62,6 @@ CPP_PROTOS := $(filter %/roachpb/metadata.proto %/roachpb/data.proto %/roachpb/i
 CPP_HEADERS := $(subst ./,$(NATIVE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.h))
 CPP_SOURCES := $(subst ./,$(NATIVE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.cc))
 
-$(PROTOC_PLUGIN):
-	$(GO_INSTALL) $(REPO_ROOT)/pkg/cmd/protoc-gen-gogoroach
-
-$(GRPC_GATEWAY_PLUGIN):
```
Comment (lego) [inline build/protobuf.mk]: It appears this plugin is still used. Is there another build rule taking care of this now? (e.g. this got replaced by a `pkg/cmd/...` command)

**Review by lgo — APPROVED** [review]
LGTM! I was just going to comment on a problem with bin files not cleaning (well, previously) but this is even better!

File: build/protobuf.mk (hunk 1 above)
Comment (benesch) [reply build/protobuf.mk]: Yeah, it's included in the $(BOOTSTRAP_TARGET) recipe, so including it here was redundant. You should definitely check my eyes on this, though.

Comment (lego) [reply build/protobuf.mk]: 👍 looks good.

**Comment by benesch** [comment]
Thanks, @lego!

**Pull request merged**